hbase_sql/src/test/java/com/splicemachine/derby/utils/VacuumIT.java

```diff
@@ -105,46 +100,77 @@
             .around(spliceTableIWatcher)
             .around(spliceTableJWatcher);
 
+    @Before
+    public void waitForStaleTransactions() throws Exception
+    {
+        // wait at max 60s for other transactions to finish
+        // we have defined SerialTest.class, so no other test should run in parallel
+        for(int i=0; i<60; i++) {
+            try (ResultSet rs = methodWatcher.executeQuery(
+                    "call SYSCS_UTIL.SYSCS_GET_ACTIVE_TRANSACTION_IDS()")) {
+                String actual = TestUtils.FormattedResult.ResultFactory.toString(rs);
+                if( actual.equals("") ) return;
+                Thread.sleep(1000);
+                System.out.println("VacuumIT: Waiting for active transactions:\n" + actual);
```

Comment:
Should this be a LOG instead of println?

Reply:
log would be only visible when logging is enabled for this class, i would rather use println here to have this prominently visible in the output.

Reply:
Logging at INFO level should be visible by default, but if it's too much of a hassle println is fine since this is only a test